integration_tests/internal/pkgr.yml

```diff
@@ -45,3 +44,5 @@ Customizations:
              - R_MAKEVARS_USER: "~/.R/Makevars_data.table"
         - internal:
             Repo: local
+        - mrgsolve:
```

Comment:
this is adding extra complexity to this 'test' which is to test local install of an internal package, should be removed/pulled to a separate test dir